pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -2120,6 +2187,62 @@ func (k kafkaManager) installJRE(ctx context.Context) error {
 	})
 }
 
+// installAzureCli installs azure cli on the kafka node which is necessary for
+// retrieving endpoint connection string for the roachtest azure event hub.
+func (k kafkaManager) installAzureCli(ctx context.Context) error {
```

Comment:
I wonder if we could simplify things. We already have `azure-cli` [1] on the node where (cdc) `Run` function is executing. Wouldn't be simpler to get the connection string there, say dump them to a flat file, and `roachprod.Put` to the kafka node?

[1] https://github.com/cockroachdb/cockroach/pull/105788

Reply:
Is the only benefit here to avoid installing azure cli on the node? The connection string has the secret key; dumping it to a flat file and `roachprod.Put` feels a bit more complicated.

Reply:
It's just adding a redundancy wrt boilerplate; maybe we could refactor it as a `roachprod install` command [1]. Otherwise, it's fine to keep the existing logic.

[1] https://github.com/cockroachdb/cockroach/blob/f428c197547b2dbf4191a0ffbe6f8f885d5329d9/pkg/roachprod/install/install.go#L23